tests/jobs/test_scheduler_job.py

```diff
@@ -1059,6 +1067,113 @@ def test_process_file_should_failure_callback(self):
             os.remove(callback_file.name)
 
 
+class DagFileProcessorQueriesCount(unittest.TestCase):
```

Comment:
Should we add `Test` as a suffix or prefix?

Reply:
Fixed.